docs/start/install.rst

```diff
@@ -66,44 +70,76 @@ The following platforms are supported:
 
   Windows wheels are available for Windows 10 or later.
 
-
+.. _Install Dev Version:
 
 Install latest dev version
 --------------------------
 
 If you wish to test the latest version of ``datatable`` before it has been
 officially released, then you can use one of the binary wheels that we build
-as part of our Continuous Integration process.
+as part of our Continuous Integration process. 
 
-If you are on Windows, then pre-built wheels are available on `AppVeyor`_.
-Click on a green main build of your choice, then navigate to the "Artifacts"
-tab, copy the wheel URL that corresponds to your Python version, and finally
-install it as:
+There are two sources available:
 
-.. code-block:: doscon
+- For macOS and Linux please use our `S3 repository`_. 
+
+- For Windows installations, you can find the latest wheels within our
+  `AppVeyor`_ build system.
+
+
+Install from the Amazon S3 Repository (macOS & Linux)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+First it is **recommended** to :ref:`Create a Python Virtual Environment<PVE Linux & macOS>`.
+
+Both the most current and historical development wheels for macOS & Linux can 
+be found at our `S3 repository`_.
 
-    C:\> pip install YOUR_WHEEL_URL
+- To install the most current development version, **scroll to the bottom** of 
+  the page to find the latest links, and then download or copy the URL of a
+  wheel that corresponds to your Python version and platform. 
 
-For macOS and Linux, development wheels can be found at our `S3 repository`_.
-Scroll to the bottom of the page to find the latest links, and then download
-or copy the URL of a wheel that corresponds to your Python version and
-platform. This wheel can be installed with ``pip`` as usual:
+  .. image:: ../_static/amazon_s3_repo.png
+    :scale: 25 %
+    :alt: datatable S3 binary wheel links
+
+- This wheel can be installed with ``pip`` as usual:
 
 .. code-block:: console
 
-    $ pip install YOUR_WHEEL_URL
+    $ pip install https://AWS S3 DATATABLE BUILD WHEEL URL.whl
 
+Install from AppVeyor (Windows)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+First it is **recommended** to :ref:`Create a Python Virtual Environment<PVE Windows>`.
 
-Build from source
------------------
+- To find the latest build wheels for datatable, go to datatable's `AppVeyor`_ 
+  CI/CD Instance and select the platform you're installing for:
```

Comment:
This will only show you the most recent build. It could be a `main` build or a PR build. It could also be a failed build. What really needs to be done is that part you have removed here:
https://github.com/h2oai/datatable/pull/3383/files#diff-0efa84325a40cac71ec0eadd2f2eec4890f28d0acac6e41a7c633d950c2d173dL78-L81